docs/tutorials/connecting-to-ev3dev-with-ssh.md

```diff
@@ -91,36 +91,48 @@ you have configured a network connection before continuing.
 
 
 *   {: tab="Windows"}
-    To connect via SSH on Windows, you will need a program called [PuTTY].
-    Download and run it from [their download page][PuTTY download]. In the
-    *PuTTY Configuration* window, type in `ev3dev` for the "host name".
-    Then click the *Open* button to connect.
-
-    {% include /util/screenshot.html source="/images/windows/10/putty-configuration-ev3dev.png" %}
+    Windows 10 now supports the good old `ssh` program. In cmd.exe, run the following command
```

Comment:
We should also say that this works in PowerShelll too. Also we should recommend using the new Windows Terminal program, otherwise things like ANSI colors and non-ASCII characters probably won't work right.

Reply:
Yup. Ssh it is installed by default and works on PowerShell. I considered saying to use the windows terminal but it is NOT installed by default.